to_fileserver.sh

```diff
@@ -1,16 +1,41 @@
 #!/bin/bash
 
 ARCH=$1
-BUILDER_NAME=$(echo $2 | sed 's,:,_,g')
```

Comment:
Builder name is builder name.
https://docs.buildbot.net/current/manual/configuration/properties.html?highlight=unique#common-build-properties
`This is a string that indicates which Builder the build was a part of. The combination of buildername and buildnumber uniquely identify a build.`

Reply:
makefile cannot handle ":" in a path.
So I have no choice to replace it. I will still be unique after.